vault/resource_consul_secret_backend_role.go

```diff
@@ -24,11 +33,11 @@ func consulSecretBackendRoleResource() *schema.Resource {
 				ForceNew:    true,
 				Description: "The name of an existing role against which to create this Consul credential",
 			},
-			"path": {
+			"backend": {
```

Comment:
If we rename this field, we'll need to go through [a renaming process](https://www.terraform.io/docs/extend/best-practices/deprecations.html#provider-attribute-rename). I'd prefer to avoid that if possible because path/backend is inconsistent in the provider and I'm thinking it would be best to take them all through the deprecation process at once. But, I may be overlooking something. What do you think?

Reply:
Hm, I think you make a valid point. That sounds reasonable to me.

Let's do it! If you're cool with starting the renaming process, that would be great.

Reply:
I think I've correctly started the renaming process here now.